### PR TITLE
Vitals design polish from 4/1/21 review

### LINKED ISSUE #985: Design polish for Vitals
**What needs to be done? Why does it need to be done?**
There are a few design details to address in the latest Vitals front-end implementation. They're outlined here: 

See mocks in Figma:
https://www.figma.com/file/H08AQJSaAIRxfeMP4iCUPh/UP-MVP?node-id=2579%3A0
https://www.figma.com/file/H08AQJSaAIRxfeMP4iCUPh/UP-MVP?node-id=2584%3A5

Ping @soniachokshi with any questions!

**Additional context**
Add any other context about the task here.

### LINKED ISSUE #1000: Loading component adds an extra semicolon to the page
**Describe the bug**
When switching between Explore and Vitals tabs, you can see a semicolon added to the DOM under the Vitals link while the BounceLoader is rendered. 

**To Reproduce**
1. Go to ND dashboard
2. Click on Vitals tab
3. See extra semicolon on the page

**Expected behavior**
Expect the loader to maintain its position at the center of the page and to not render an extra semicolon character.

**Screenshots**
Screen Shot 2021-04-09 at 9 50 09 AM

**Desktop (please complete the following information):**
 - OS [e.g. Windows 10]: 
 - Browser [e.g. chrome, edge]: 
 - Version [e.g. 22]: 

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]: 
 - OS: [e.g. iOS8.1]: 
 - Browser [e.g. stock browser, safari]: 
 - Version [e.g. 22]: 

**Additional context**
[BounceLoader package](https://github.com/davidhu2000/react-spinners)

I think we're just missing semicolons in the override CSS, but when I tried to just add those in the position of the loader changed, so we'll need to figure out the correct styles.

### PR DESCRIPTION
**Pull request by daschi**
## Description of the change

This updates some of the styles on the vitals dashboard based on the 4/1 design review. 

Preview: https://recidiviz-dashboard-stag-e1108--vitals-nn6l81bf.web.app/community/vitals
TODO 
- [x] Send preview app to Sonia

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #985 
> Closes #1000 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/CoreNavigation.scss [hunk 1]
```diff
@@ -36,41 +36,63 @@
     }
   }
 
-  &__left {
-    padding-left: 4rem;
-    display: flex;
-  }
-
-  div.CoreNavigation__left {
+  .CoreNavigation__left {
```
Comment (jovergaag) [inline src/core/CoreNavigation.scss:39]: Why not `&__left`?

File: src/core/CoreNavigation.scss [hunk 2]
```diff
@@ -36,41 +36,63 @@
...
+      &.show {
+        background-color: $core-dropdown-color;
+        border-radius: 1.66rem;
+      }
+
+      .show {
+        padding: 0.62rem 0;
+        border-radius: 0 0 1.66rem 1.66rem;
+        box-shadow: $inset-shadow-30;
+      }
```
Comment (jovergaag) [inline src/core/CoreNavigation.scss:72]: Can these two be collapsed?

File: src/core/CoreNavigation.scss [hunk 3]
```diff
@@ -142,7 +142,7 @@
 
       > a,
       button {
-        color: $default-text-color;
+        // color: $default-text-color;
```
Comment (jovergaag) [inline src/core/CoreNavigation.scss:145]: Did you mean to remove this? And the font-size on line 154?

File: src/core/CoreNavigation.scss [hunk 4]
```diff
@@ -103,33 +124,12 @@
...
-        transform: scale(1, 1);
-        min-width: 11.81rem;
-        background-color: $core-dropdown-color;
-      }
     }
   }
 
   &__right {
     align-items: center;
+
```
Comment (jovergaag) [inline src/core/CoreNavigation.scss:132]: Extra line here.

File: src/core/VitalsSummaryTable/VitalsSummaryTable.scss [hunk 5]
```diff
@@ -209,10 +206,15 @@ $border-styles: 1px solid $slate-20;
     }
   }
 
+  /* All sortable headers */
```
Comment (jovergaag) [inline src/core/VitalsSummaryTable/VitalsSummaryTable.scss:209]: Thanks for all these comments. Much easier to grok

**Review by jovergaag — CHANGES_REQUESTED** [review]
A few little tidying up things. But nothing show stopping!

**Review by jovergaag — APPROVED** [review]

File: src/core/CoreNavigation.scss (hunk 2 above)
Comment (daschi) [reply src/core/CoreNavigation.scss:72]: Ooh, I was able to collapse the one on line 68 with the style on line 83, good eye. The style on line 63 refers to a parent selector, but I think moving line 68 down makes that more clear.

**Pull request merged**